### DISCUSSION TITLE
RabbitMQ dashboard gone?

### DISCUSSION THREAD

**Discussion by SalomonHenke** [microsoft/aspire#2407] [Q&A] [answered]
Fair warning, newbie question.
Why can't i access the rabbitmq dashboard any longer? 
![image](https://github.com/dotnet/aspire/assets/96744651/61ce7ab9-a82d-4d36-86d9-590bffd9e7f8)
Endpoint none.
Port:
![image](https://github.com/dotnet/aspire/assets/96744651/500bd82d-5ba8-4b70-b256-9c26aef1620a)
![image](https://github.com/dotnet/aspire/assets/96744651/00adad3e-1c46-47d1-801b-2cce6e69c509)

But! 
Messaging works fine. I do wonder if it has anything to do with my masstransit configuration?
![image](https://github.com/dotnet/aspire/assets/96744651/7498a5dc-0e04-4b73-9dee-633c340b2db2)
![image](https://github.com/dotnet/aspire/assets/96744651/534265b7-3fee-44d5-a9b9-38cac8cdf705)

Would be nice to validate exchanges and such. 

Thanks in advance.

**Comment by davidfowl** [comment, marked as answer]
Yes, we removed it in preview3, it'll back in a future preview (5+), we have to work out some things with respect to multiple endpoints on containers. 

You can add the endpoint back via:

```C#
builder.AddRabbitMqContainer("rabbitmq")
.WithAnnotation(new ContainerImageAnnotation { Image = "rabbitmq:3-management" })
.WithEndpoint(name: "management", scheme: "http", containerPort: 15672);
```

**Reply by SalomonHenke** [reply]
Thanks :)

**Reply by davidfowl** [reply]
cc @mitchdenny @timheuer Related to our discussion about the rabbitmq image.
